bin/varnishd/http2/cache_http2_deliver.c

```diff
@@ -241,6 +241,9 @@ h2_build_headers(struct vsb *resp, struct req *req)
 		r = strchr(hp->hd[u].b, ':');
 		AN(r);
 
+		if (http_IsFiltered(hp, u, HTTPH_C_SPECIFIC))
+			continue; //rfc7540,l,2999,3006
```

Comment:
Wouldn't it be better to reference sections instead of line numbers? When looking up RFCs online they usually don't show line numbers.

Reply:
true, but we had that format defined before

Reply:
I don't disagree but this is how we get coverage:

https://varnish-cache.org/rfc/rfc7540.html

And this doesn't even cover the entire section.